Architecture.md

```diff
@@ -71,6 +71,79 @@
 
 ***TODO: [GIT] how are the projects stored and how is authentication handled upon push***
 
+As all PaaS providers, PaaSTech needs to store the code of the projects created by its clients. This is done using a Git repository, which is created upon project creation. The repository is then exposed to the client, who can push its code to it.
```

Comment:
Please use `Client` instead of `client`, as it refers to the Definitions. `its` => `their`, it's the Clients' code and Clients are not objects.

```suggestion
As all PaaS providers, PaaSTech needs to store the code of the projects created by its Clients. This is done using a Git repository, which is created upon project creation. The repository is then exposed to the Client, who can push their code to it.
```